chat-plugins/info.js

```diff
@@ -1337,6 +1337,18 @@ exports.commands = {
 		if (showAll || target === 'tiering' || target === 'tiers' || target === 'tier') {
 			buffer.push("<a href=\"https://www.smogon.com/ingame/battle/tiering-faq\">Tiering FAQ</a>");
 		}
+		if (showAll || toId(target).startsWith('lostpass') || target === 'pwreset') {
+			buffer.push("PM a member of Global Staff (@ in the Help room) to facilitate password resets.");
+		}
+		if (showAll || toId(target) === 'lostteams' || toId(target) === 'deletedteams') {
+			buffer.push("Have you cleared your cookies? This deletes your teams. You may  have done this accidentally when clearing your history, or by using  private browsing. Always back up your teams to a pastebin!");
+		}
+		if (showAll || target === 'ladder') {
+			buffer.push("Ladder battles are any matches you start with the 'Battle!' button besides unrated random battles.");
+		}
+		if (showAll || toId(target).startsWith('minimod')) {
+			buffer.push("Minimodding is when someone is moderating the chat when it isn't their job to do so - this is against the rules. This can include telling people they'll be muted, asking for people to be muted, or talking about whether or not people should be muted.");
```

Comment:
I think as many users who don't know what autoconfirmed is also don't know what minimoddding is